mentoring/how_to_give_great_feedback.md

```diff
@@ -86,5 +86,5 @@ As you gain experience mentoring, you will get a feel for the different ways stu
 ## Unsure of something? Chat to other mentors?
 
 One of the best things about being an Exercism mentor is the community of other mentors you can learn from.
-If you've got a question about a solution, or just want to bounce ideas with other mentors, jump into our [Slack Workspace](https://exercism-team.slack.com/) (you should receive an invite when you sign up to mentor) and ask away!
+If you've got a question about a solution, or just want to bounce ideas with other mentors, jump into our [Slack Workspace](https://exercism-team.slack.com/) (Send a request to be added to the workspace to hello@exercism.org) and ask away!
```

Comment:
Could you revert this change? We really don't want people to send us emails.

Reply:
For sure. :)